keps/sig-testing/2420-reducing-kubernetes-build-maintenance/kep.yml

```diff
@@ -16,7 +16,7 @@ approvers:
   - justaugustus
 # NOTE: there's no production change in this KEP
 prr-approvers:
-  - TBD
+  - johnbelamaric
```

Comment:
@johnbelamaric done.
still looking for why it apparently doesn't parse.

Reply:
kep.yml -> kep.yaml

Reply:
john found it: https://github.com/kubernetes/enhancements/pull/2469/commits/58ed75f953981e71564b5c7e54be64dad1ddd4c1